sdk/core/core-http/test/policies/bearerTokenAuthenticationPolicyTests.ts

```diff
@@ -11,13 +11,16 @@ import { HttpOperationResponse } from "../../src/httpOperationResponse";
 import { HttpHeaders } from "../../src/httpHeaders";
 import { WebResource } from "../../src/webResource";
 import { BearerTokenAuthenticationPolicy } from "../../src/policies/bearerTokenAuthenticationPolicy";
-import {
-  ExpiringAccessTokenCache,
-  TokenRefreshBufferMs
-} from "../../src/credentials/accessTokenCache";
+import { ExpiringAccessTokenCache } from "../../src/credentials/accessTokenCache";
 import { AccessTokenRefresher } from "../../src/credentials/accessTokenRefresher";
 
-describe("BearerTokenAuthenticationPolicy", function() {
+// Token is refreshed one millisecond BEFORE it expires.
```

Comment:
this isn't one millisecond, this is one second (1000 ms)

Reply:
Resolving since I reverted this.

Reply:
Comment still seems to be here and not make sense?

Reply:
This comment does make sense in this current state of the code. The token is indeed refreshed this time before its expiration date. In fact is forcefully refreshed 😅 The new code in core-https will return the valid token during this period while it tries to refresh it in the background.